koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -383,24 +385,40 @@ def _write_demultiplexed_logs(self, build_id, logs_dir):
             raise ContainerError(msg)
         platform_logs = {}
         for entry in logs:
-            platform = entry.platform
+            platform, line = entry.platform, entry.line
             if platform == METADATA_TAG:
-                meta_file = entry.line
+                meta_file = line
                 source_file = os.path.join(koji.pathinfo.work(), meta_file)
                 uploadpath = os.path.join(logs_dir, os.path.basename(meta_file))
                 shutil.copy(source_file, uploadpath)
                 continue
 
+            if self._user_warnings.is_user_warning(line):
+                self._user_warnings.store(line)
+                continue
```

Comment:
shouldn't we still save it also to corresponding log file? orchestrator or worker ones

Reply:
I suppose we shouldn't because it'd be the redundancy of data

Reply:
I'd agree here with serhii here

Reply:
ok